contentcuration/contentcuration/frontend/shared/views/StudioBanner.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+
+  <div
+    class="banner notranslate"
+    data-testid="studio-banner"
+    :style="{ backgroundColor: error ? $themePalette.red.v_100 : '' }"
+  >
+    {{ errorText }}
+  </div>
+
+</template>
+
+
+<script>
+
+  import useKLiveRegion from 'kolibri-design-system/lib/composables/useKLiveRegion';
+
+  export default {
+    name: 'StudioBanner',
+    setup() {
+      const { sendPoliteMessage } = useKLiveRegion();
+      return { sendPoliteMessage };
+    },
+    props: {
+      error: {
+        type: Boolean,
+        default: false,
+      },
+      errorText: {
+        type: String,
+        default: '',
+      },
+    },
+    watch: {
+      errorText(newText) {
+        if (newText.length && this.error) {
+          this.sendPoliteMessage(newText);
+        }
+      },
+    },
+    mounted() {
+      if (this.errorText.length && this.error) {
```

Comment:
The logic here is identical to the logic in the `watch` block. this code duplication could be avoided by refactoring it into a single method